### PR TITLE
[OSDEV-716] Search. Lost refresh icon.

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-716](https://opensupplyhub.atlassian.net/browse/OSDEV-716)

* The refresh icon has been made visible.

### CODE REVIEW

File: src/app/src/components/FilterSidebarSearchTab.jsx [hunk 1]
```diff
@@ -174,10 +175,10 @@ function FilterSidebarSearchTab({
             onClick={() => resetFilters(embed)}
             disableRipple
             className={classes.reset}
-            color="secondary"
+            color="primary"
```
Comment (roman-stolar) [inline src/app/src/components/FilterSidebarSearchTab.jsx:178]: Why you change color?

Comment (Innavin369) [reply src/app/src/components/FilterSidebarSearchTab.jsx]: changed

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Pull request merged**